enhance(test-cli-execute): Many Improvements to `execute remote` command (#1822)
ethereum/execution-specs: docs/running_tests/execute/hive.md

## The `eest/execute-blobs` Simulator

The `blob_transaction_test` execute test spec sends blob transactions to a running client. Blob transactions are fully supported in execute mode:

- Blob transactions can be sent via `eth_sendRawTransaction`
- Blob validation via `engine_getBlobsVX` endpoints (when Engine RPC available)

> - Automatic gas pricing for blob gas fees

Reviewer: ```suggestion
- Automatic gas pricing is used for the blob gas fees
```

Tests can be run using:

```bash

> ./hive --client besu --client-file ./configs/osaka.yaml --sim ethereum/eest/execute-blobs

Reviewer: ```suggestion
./hive --client besu --client-file ./configs/osaka.yaml --sim ethereum/eels/execute-blobs
```

Reviewer: We also need to merge this: https://github.com/ethereum/hive/pull/1365

```

> **Note**: If Engine RPC is unavailable, blob transactions will be sent but `getBlobsV*` validation is skipped.

Reviewer: ```suggestion
**Note**: If the Engine RPC is unavailable, blob transactions will be sent and `getBlobsV*` validation is skipped.
```
